cli/deployment/config.go

```diff
@@ -134,24 +135,52 @@ func (c Config) YAML(outputs map[string]map[string]interface{}) ([]byte, error)
 
 // Function findAllDependencies finds all dependencies base on cross-deployment references found in config YAML,
 // if no dependencies found, returns empty slice.
-func (c Config) findAllDependencies(configs map[string]Config) []Config {
+func (c Config) findAllDependencies(configs map[string]Config) ([]Config, error) {
 	var result []Config
 	refs := c.findAllOutRefs()
 	dependencies := map[string]Config{}
 	for _, ref := range refs {
 		fullName, _, _ := c.parseOutRef(ref)
 		dependency, found := configs[fullName]
+		exists := false
 		if !found {
-			log.Fatalf("Could not find config for deployment = %s", fullName)
+			projectAndName := strings.Split(fullName, ".")
+			deployment := &Deployment{
+				config: Config{
+					Project: projectAndName[0],
+					Name:    projectAndName[1],
+				},
+			}
+			status, err := GetStatus(deployment)
+
+			if err != nil {
+				log.Printf("GetStatus for deployment = %s", fullName)
+				return nil, err
+			}
+			switch status {
+			case Done:
+				exists = true
+			case NotFound:
+				message := fmt.Sprintf("Could not find config or existing deployment = %s", fullName)
```

Comment:
Is this meant to be "**for** existing deployment?"

Reply:
not typo, with CLI you can pass set of yaml files and set dependency between yaml files (config) or reference existing deployment created before